### PR TITLE
Init the client twoice,Make sure the config take effect

### PR DESCRIPTION
**Pull request by QuanjieDeng**
This may not be the optimal solution, but at present I can only modify it like this

### CODE REVIEW

File: src/agent/protocol/grpc/clients/HeartbeatClient.ts [hunk 1]
```diff
@@ -47,6 +47,13 @@ class HeartbeatClient implements Client {
   }
 
   start() {
+    /*
+    init  the heartbeatClient again
+    */
```
Comment (kezhenxu94) [inline src/agent/protocol/grpc/clients/HeartbeatClient.ts:52]: Remove this comment and the initialization in the `constructor`

File: src/agent/protocol/grpc/clients/TraceReportClient.ts [hunk 2]
```diff
@@ -50,6 +50,14 @@ class TraceReportClient implements Client {
   }
 
   start() {
+    /*
+    init the reporterClient again
+    */
```
Comment (kezhenxu94) [inline src/agent/protocol/grpc/clients/TraceReportClient.ts:55]: Remove this comment and the initialization in the `constructor`

**Comment by kezhenxu94** [comment]
Another workaround is to set the config via environment variables

**Comment by kezhenxu94** [comment]
Hey @QuanjieDeng , why do you close the PR, the fix is correct, I just wanted you to remove the duplicated initialization in the constructors

**Comment by QuanjieDeng** [comment]
> Hey @QuanjieDeng , why do you close the PR, the fix is correct, I just wanted you to remove the duplicated initialization in the constructors

yes,Im not good at TypeScript,Im trying  fix this  bug

**Comment by kezhenxu94** [comment]
> > Hey @QuanjieDeng , why do you close the PR, the fix is correct, I just wanted you to remove the duplicated initialization in the constructors
> 
> yes,Im not good at TypeScript,Im trying fix this bug

It's OK

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**